ui/src/components/SubDescription/SubDescription.tsx

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable react/no-array-index-key */
+import React from 'react';
+import styled from 'styled-components';
+import { z } from 'zod';
+import { variables } from '@splunk/themes';
+import Link from '@splunk/react-ui/Link';
+import { SubDescriptionSchema } from '../../types/globalConfig/pages';
+
+export const SubTitleComponent = styled.p`
+    & {
+        font-size: ${variables.fontSize};
+        margin-bottom: 10px;
+        white-space: pre-line;
+    }
+`;
+
+type SubDescriptionProps = z.TypeOf<typeof SubDescriptionSchema>;
+
+const mapTextToElements = (props: SubDescriptionProps) => {
+    const splitedtextBySlugs: string[] | undefined = props?.text.split(/\]\]|\[\[/);
+
+    return splitedtextBySlugs
+        ?.map((text) => {
+            const linkToReplace = props?.links?.find((link) => link.slug === text);
+
+            if (linkToReplace) {
+                return (
+                    <Link to={linkToReplace.link} target="_blank" rel="noreferrer">
```

Comment:
https://splunkui.splunk.com/Packages/react-ui/Link?section=develop
There a special prop for that
```suggestion
                    <Link to={linkToReplace.link} openInNewContext >
```